src/Javis.jl

```diff
@@ -134,6 +135,10 @@ function centered_point(pos::Point, width::Int, height::Int)
     Point(pos.x - width / 2, pos.y - height / 2)
 end
 
+function centered_point(pos::DelayedPosition, width::Int, height::Int)
```

Comment:
A small docstring would be great. Also does width and height need to be `Int64`?